applications/vessel-history/src/features/search/search.thunk.ts

```diff
@@ -111,40 +111,35 @@ export type VesselSearchThunk = {
   advancedSearch?: Record<string, any>
 }
 
-const trackData = (
-  query: any,
-  results: SearchResults | null,
-  actualResults: number
-) => {
+const trackData = (query: any, results: SearchResults | null, actualResults: number) => {
   if (!query.offset || query.offset === 0) {
-    const vessels = results?.vessels.slice(0, 5).map(vessel => {
+    const vessels = results?.vessels.slice(0, 5).map((vessel) => {
       return {
         gfw: vessel.id,
-        tmt: vessel.vesselMatchId
+        tmt: vessel.vesselMatchId,
       }
     })
     uaEvent({
       category: 'Search Vessel VV',
       action: 'Click Search',
       label: JSON.stringify({ ...query, vessels }),
-      value: results?.total
+      value: results?.total,
     })
   } else {
     uaEvent({
       category: 'Search Vessel VV',
       action: 'Click Load More',
       label: actualResults.toString(),
-      value: results?.total
+      value: results?.total,
```

Comment:
not directly related with the review but aren't we running the pre-commit hook to lint the files in this project? o how did we end up having this in the repo?

Reply:
Yes we are, but it might be not too strict... I have my IDE set to autoformat on save but maybe some of us does not have eslint plugin properly configured to do so.
We should probably double check and fix the linting rules to  be in sync between the [pre-commit hook settings](https://github.com/GlobalFishingWatch/frontend/blob/develop/package.json#L90-L102) and the [autoformat on save](https://github.com/GlobalFishingWatch/frontend/blob/develop/.vscode/settings.json#L17-L18)

Reply:
yes I have the same configuration and taking a look at the configuration seems to be ok.. lets see if that happens again